relay-metrics/src/aggregation.rs

```diff
@@ -841,6 +847,23 @@ pub struct AggregatorConfig {
     ///
     /// Defaults to `200` bytes.
     pub max_tag_value_length: usize,
+
+    /// Maximum amount of bytes used for metrics aggregation.
+    ///
+    /// When aggregating metrics, Relay keeps track of how many bytes a metric takes in memory.
+    /// This is only an approximation and does not take into account things such as pre-allocation
+    /// in hashmaps.
+    ///
+    /// Defaults to `None`, i.e. no limit.
+    pub max_total_bucket_bytes: Option<usize>,
+
+    /// Maximum amount of bytes used for metrics aggregation per project.
+    ///
+    /// Similar measuring technique to `max_total_bucket_bytes`, but instead of a
+    /// global/process-wide limit, it is enforced per project id.
```

Comment:
```suggestion
    /// global/process-wide limit, it is enforced per project key.
```

Reply:
Can we still fix this and also use a better config name?

Reply:
What name would you suggest? `max_bucket_bytes_total` and `max_bucket_bytes_per_project_key`?